### PR TITLE
Adding recipe for gregorio-mode

### PR DESCRIPTION
**Pull request by cajetanus**
Just adding a recipe for gregorio mode, available on github.

### CODE REVIEW

File: recipes/gregorio-mode [hunk 1]
```diff
@@ -0,0 +1,2 @@
+(gregorio-mode :fetcher github
+	       :repo "cajetanus/gregorio-mode")
```
Comment (syohex) [inline recipes/gregorio-mode]: Typo ? `gregorio-mode` -> `gregorio-mode.el`.

**Comment by cajetanus** [comment]
Sorry about that, forgot the .el at the end.

**Comment by syohex** [comment]
Your package does not follow Emacs packaging convention.

Please see
- https://www.gnu.org/software/emacs/manual/html_node/elisp/Simple-Packages.html
- https://www.gnu.org/software/emacs/manual/html_node/elisp/Packaging-Basics.html

Example
- https://github.com/milkypostman/melpa/blob/master/package-build.el#L1-L40 (header part)
- https://github.com/milkypostman/melpa/blob/master/package-build.el#L1427 (footer part)

**Comment by purcell** [comment]
Yep. Additional feedback:

- "C-c u" is reserved for users (see the [keybinding conventions](https://www.gnu.org/software/emacs/manual/html_node/elisp/Key-Binding-Conventions.html)) so you can't use it here
- Consider adding `;;;###autoload` cookies for the major mode definition form and the form which extends `auto-mode-alist`.

When fixed, please open a fresh PR with a single recipe commit. Note that you can test it locally before submitting by following the directions in the README. :-)

**Pull request closed without merging**